stream_alert/alert_processor/main.py

```diff
@@ -46,350 +42,87 @@ def handler(event, context):
 
         message = sns_payload['Message']
         try:
-            alerts = json.loads(message)
-            if 'default' in alerts:
-                StreamOutput(context).run(alerts)
-            else:
-                logging.info('Unexpected message: %s', alerts)
+            alert = json.loads(message)
         except ValueError as err:
-            logging.error('An error occured while decoding message to JSON: %s', err)
-
-class OutputRequestFailure(Exception):
-    pass
-
-class StreamOutput(object):
-    """Route StreamAlerts to their declared outputs.
-
-    Attributes:
-        creds [dict]: Store decrypted credentials
-            for StreamAlert outputs.  Dict keys are the name
-            of the output, and their value is a named tuple
-            containing the (url, secret) to send the message.
-
-        bucket [string]: the S3 bucket to store alerts.
-
-        lambda_region [string]: the region of the currently executing
-            lambda function.
-
-    Public Methods:
-        run
-        emit_cloudwatch_metrics
-    """
-    def __init__(self, context):
-        self.creds = {}
-        self.lambda_region = self._get_region(context)
-        self.bucket = self._get_bucket_name(context)
-
-    def run(self, alerts):
-        """Send an Alert to its described outputs.
+            logging.error('an error occurred while decoding message to JSON: %s', err)
+            return
```

Comment:
Do we not want to bubble this error all the way up so that the Lambda invocation fails?